Scripts/PowerUps.cs

```diff
@@ -232,4 +233,258 @@ public void Apply(Player playerWhoSelected, Player otherPlayer)
       playerWhoSelected.Gun.BulletGravity *= 2; // Bullets drop faster
     }
   }
+
+  public class WallSpider : IPowerUp
+  {
+    // Walljumping briefly increases movement speed
+    public string Name => "Wall Spider";
+    public Rarity Rarity => Rarity.Common;
+    private ulong _msecSinceLastWallJump;
```

Comment:
Since all powerups only have one instance, they share this field, so the power up behaviours will be weird if both players have the same power up

Reply:
As this will require some relatively large changes, I can see us approving this PR and fix this separately